src/domains/scrobbleAlbum/partials/Tracklist.tsx

```diff
@@ -247,11 +258,47 @@ export default function Tracklist({ albumInfo, tracks }: { albumInfo: Album | nu
         noMenu
         analyticsEventForScrobbles="Scrobble individual album song"
         scrobbles={tracks || []}
+        scrobblesRemovalPattern={useRemovalPattern}
         onSelect={toggleSelectedTrack}
         selected={selectedTracks}
       >
         <EmptyDiscMessage />
       </ScrobbleList>
+
+      <div className="d-flex flex-column mt-4 album-removal-pattern">
+        <FormGroup>
+          <div className="col-12">
+            <label htmlFor="albumRemovalPattern" className="mb-2">
+              <Trans i18nKey="albumRemovalPattern" />:
+            </label>
+            <FontAwesomeIcon
+              id="removalPatternInfoIcon"
+              className="px-3"
+              icon={faQuestionCircle}
+              color="var(--bs-gray)"
+              onClick={toggleRemovalPatternCopy}
+            />
+          </div>
+          <div className="col-6">
+            <Input
+              className="form-control-sm form-control"
+              id="albumRemovalPattern"
+              onChange={handleRemovalPatternChange}
+            />
+          </div>
+          <div className="col-12">
+            <Alert
+              color="dark"
+              isOpen={showRemovalPatternCopy}
+              toggle={toggleRemovalPatternCopy}
+              className="text-justify mt-3"
+              fade={false}
+            >
+              <Trans i18nKey="albumRemovalPatternDescription" />
+            </Alert>
+          </div>
+        </FormGroup>
+      </div>
```

Comment:
I'm wondering if this could be another partial instead of being part of `Tracklist.tsx`.

All this could become something like:

```
      <TracklistCleanupForm onChange={handleRemovalPatternChange} />
```

What do you think?

Reply:
I'll check it out as soon as I'm more rested, I'm at the airport right now, after a wedding party. I'll come back to you. But, I believe it can make sense.